package.json

```diff
@@ -91,8 +91,6 @@
     "@typescript-eslint/parser": "^4.3.0",
     "auto-dist-tag": "^1.0.0",
     "aws-sdk": "^2.686.0",
-    "babel-eslint": "^10.1.0",
-    "babel-plugin-module-resolver": "^4.0.0",
```

Comment:
👋

Reply:
Ya, this is good to remove. IIRC, we leverage `ember-cli-babel` to do our transpilation now and it handles this resolution for us.